From 00354a8e4d1270fae6ea733564f5ed9033ac9991 Mon Sep 17 00:00:00 2001
From: "Radu Pirea (NXP OSS)" <radu-nicolae.pirea@oss.nxp.com>
Date: Thu, 22 Jul 2021 16:53:37 +0300
Subject: [PATCH 14/16] s32gen1image: qspi micron cfg: fix dllcr value

The previews value(0x82860704) for DLLCR didn't worked on S32G rev 2.
The BlueBox 3 board was unable to boot from QSPI.
Using 0x42860704 the board boot from QSPI.

Issue: ALB-7466

Upstream-Status: Pending

Signed-off-by: Radu Pirea (NXP OSS) <radu-nicolae.pirea@oss.nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 tools/s32gen1image_qspi_micron.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/s32gen1image_qspi_micron.c b/tools/s32gen1image_qspi_micron.c
index fe1eac2081..012ddb6470 100644
--- a/tools/s32gen1image_qspi_micron.c
+++ b/tools/s32gen1image_qspi_micron.c
@@ -12,7 +12,7 @@
 	defined(CONFIG_TARGET_S32G274ABLUEBOX3)
 void adjust_qspi_params(struct qspi_params *qspi_params)
 {
-	qspi_params->dllcr = 0x82860704;
+	qspi_params->dllcr = 0x42860704;
 	qspi_params->dll_slv_upd_en = 0x01;
 	qspi_params->sflash_clk_freq = 133;
 }
@@ -22,7 +22,7 @@ static struct qspi_params s32g2xx_qspi_conf = {
 	.mcr      = 0x010f00cc,
 	.flshcr   = 0x00010303,
 	.bufgencr = 0x00000000,
-	.dllcr    = 0x82860704,
+	.dllcr    = 0x42860704,
 	.paritycr = 0x00000000,
 	.sfacr    = 0x00000000,
 	.smpr     = 0x44000000,
-- 
2.17.1

